### DISCUSSION TITLE
version 3.0 or .net 6

### DISCUSSION THREAD

**Discussion by elhajjh** [fullstackhero/blazor-starter-kit#399] [Q&A]
When will version 3.0 will be ready? or when will migration to .net 6 will be ready?

Thank you for your great effort.

**Comment by jeremiedevos** [comment]
In case you did not yet notice, the project is now updated to .net 6.

**Reply by elhajjh** [reply]
Hi, what about version 3.0,
any release time?

**Reply by ghost** [reply]
@jeremiedevos The project is updated to .net 6? I'm not seeing that it has been. When I did a new install, it was .net 5.

**Reply by jeremiedevos** [reply]
Hi @AdamsTechPros,
Well this github repository is dot net 6. Not the NuGet-package and stuff. So you will have to clone this repo and move from there. Not sure if the owner of this repo will update the NuGet.

**Reply by ghost** [reply]
Thank you for your response. I followed the instructions on the page but guess I will have to start all over after hours of configuration.

**Comment by ghost** [comment]
Did the v 3.0 update fall off the map or is that still something coming? Also where's the best place to go and ask for help with using BlazorHero?
